### PR TITLE
Auth FLAC: Extended JWT

### PR DESCRIPTION
**Pull request by amarjandu**
adds `permissions`,`groups`,`roles` into the JWT under a claim called `${OIDC_AUDIENCE}/auth0`

### CODE REVIEW

File: infra/auth/main.tf [hunk 1]
```diff
@@ -66,12 +66,18 @@ function (user, context, callback) {
       });
 
     context.accessToken[namespace + 'email']= user.email;
+    context.accessToken[namespace + 'auth0'] = {
```
Comment (chmreid) [inline infra/auth/main.tf:69]: niiiiiiiiccceeee

File: README.md [hunk 2]
```diff
@@ -2,12 +2,26 @@
...
 
-1. configure a tenant with Auth0
-1. set the secret with the Auth0 Management API in the aws secret-store [Secret Setup](#Secret-Setup)
-1. deploy infra -> sets up all the stuff. more here later. 
+### Setup/Deploy Steps
 
+1. Install Terraform provider: [see info below](#Terraform-Installation) 
+1. Configure a tenant with Auth0
+1. Update `$DSS_AUTH_HOME/environment` values with information from Tenant
+1. Set the secret with the Auth0 Management API in the aws secret-store [Secret Setup](#Secret-Setup)
```
Comment (chmreid) [inline README.md]:
```suggestion
1. Set the auth tenant secret with the Auth0 Management API in the AWS secrets store [Secret Setup](#Secret-Setup)
```

**Review by chmreid — APPROVED** [review]
1 smol suggestion

**Comment by amarjandu** [comment]
always trying to get partial credit, :p

**Pull request merged**